notebook_examples/load_data-object_storage-hive-autonomous-database.ipynb

```diff
@@ -40,6 +40,13 @@
     "---\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Compatible conda pack: [General Machine Learning](https://docs.oracle.com/en-us/iaas/data-science/using/conda-gml-fam.htm) for CPU on Python 3.8 (version 1.0)"
```

Comment:
this link (https://docs.oracle.com/en-us/iaas/data-science/using/conda-gml-fam.htm) takes you not to a 3.8 conda but only 3.7 - seems like the wrong link

should be: 

https://docs.oracle.com/en-us/iaas/data-science/using/conda-generalml-fam.htm